README.md

```diff
@@ -2,7 +2,7 @@
 
 This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## bookstore
+## Book store
```

Comment:
- [ ] Please remove all generated readme file details and additional details about the project.